test/suites/zombie_data_preservers/test_zombie_data_preservers.ts

```diff
@@ -208,12 +214,25 @@ describeSuite({
                 console.log((await customHttpProvider.getNetwork()).chainId);
 
                 const signer = new ethers.Wallet(BALTATHAR_PRIVATE_KEY, customHttpProvider);
-                const tx = await signer.sendTransaction({
-                    to: CHARLETH_ADDRESS,
-                    value: parseUnits("0.001", "ether"),
-                });
 
-                await customHttpProvider.waitForTransaction(tx.hash);
+                for (let i = 0; i <= 5; i++) {
```

Comment:
Why don't we just increase a timeout to (60_000 x 5), instead of waiting 5 times and then failing? 
Something like:
```
try {
  const nonce = await customHttpProvider.getTransactionCount(BALTATHAR_ADDRESS);
  const tx = await signer.sendTransaction({
    to: CHARLETH_ADDRESS,
    value: parseUnits("0.001", "ether"),
    nonce,
  });

  await customHttpProvider.waitForTransaction(tx.hash, 1, 60_000 * 5);
} catch (e) {
  console.log("tx inclusion failed: ", e);
  expect.fail("failed to send tx");
}


```

Reply:
We need to send the transaction again, if not it will never get included. I will add a comment to explain this as it's not obvious